qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -82,13 +82,19 @@ def qapplication():
 # -----------------------------------------------------------------------------
 # Importing resources loads the data in
 from workbench.app.resources import qCleanupResources  # noqa
-atexit.register(qCleanupResources)
 
-SPLASH = QSplashScreen(QPixmap(':/images/MantidSplashScreen.png'),
-                       Qt.WindowStaysOnTopHint)
+atexit.register(qCleanupResources)
+width = QApplication.desktop().availableGeometry().width()
+
+if width > 2048:
```

Comment:
As the only variable is the image name is it worth not duplicating the `QSplashScreen` call and defining a function to return the appropriate image?